backend/src/middleware/validation/validationMiddleware.spec.js

```diff
@@ -119,20 +118,30 @@ beforeEach(async () => {
   moderatingUser = users[1]
   commentingUser = users[2]
   const posts = await Promise.all([
-    factory.create('Post', {
-      id: 'offensive-post',
-      authorId: 'moderating-user',
-    }),
-    factory.create('Post', {
-      id: 'post-4-commenting',
-      authorId: 'commenting-user',
-    }),
+    Factory.build(
+      'post',
+      {
+        id: 'offensive-post',
+      },
+      {
+        authorId: 'moderating-user',
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-04T11:17:05Z" title="Tuesday, February 4th 2020, 12:17:05 pm +01:00">Feb 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -119,20 +118,30 @@ beforeEach(async () => {
   moderatingUser = users[1]
   commentingUser = users[2]
   const posts = await Promise.all([
-    factory.create('Post', {
-      id: 'offensive-post',
-      authorId: 'moderating-user',
-    }),
-    factory.create('Post', {
-      id: 'post-4-commenting',
-      authorId: 'commenting-user',
-    }),
+    Factory.build(
+      'post',
+      {
+        id: 'offensive-post',
+      },
+      {
+        authorId: 'moderating-user',
```


I know this isn't your code, but it doesn't really make sense, oder?
the person who wrote an offensive post is the moderating user? what they are moderating themselves?